### ISSUE TITLE
Style API docs

### ISSUE THREAD

**Issue by minkimcello** [thefrontside/effection#449]
Sort of related to #396

We need to style our API docs. I've added a base style template for typedocs _inside_ `./website/typedoc-theme/`.

**Comment by minkimcello** [comment]
I think we can add the styles as a separate PR once #396 is merged and doesn't necessarily need to happen at the same time (even though it's hideous) because it's unlikely for anyone to actually see it.

**Comment by jnicklas** [comment]
The styles we have now are good enough, let's close this!
